classes/auth/login/simpleauth.php

```diff
@@ -124,17 +124,24 @@ public function validate_user($username_or_email = '', $password = '')
 			return false;
 		}
 
-		$password = $this->hash_password($password);
 		$user = \DB::select_array(\Config::get('simpleauth.table_columns', array('*')))
-			->where_open()
-			->where('username', '=', $username_or_email)
```

Comment:
did you mean to remove this line? the or_where on the next line doesn't seem right...

Reply:
resolved